packages/calcite-components/src/components/alert/alert.e2e.ts

```diff
@@ -5,6 +5,8 @@ import { getElementXY, skipAnimations } from "../../tests/utils";
 import { openClose } from "../../tests/commonTests";
 import { CSS, DURATIONS } from "./resources";
 
+const animationDurationInMs = 400;
```

Comment:
If this is for the auto-close duration, can we rename it to convey this? I believe the open/close animations are all properly wired up, so the relevant events can be awaited on.

Reply:
fixed!